src/time_integration/paired_explicit_runge_kutta/methods_PERK3.jl

```diff
@@ -0,0 +1,386 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+using NLsolve: nlsolve
+using DelimitedFiles: readdlm
+
+@muladd begin
+#! format: noindent
+
+# Initialize Butcher array abscissae c for PERK3 based on SSPRK33 base method
+function c_PERK3_SSP33(num_stages, c_s2)
+    c = zeros(num_stages)
+
+    # Last timesteps as for SSPRK33
+    c[num_stages] = 0.5
+    c[num_stages - 1] = 1
+
+    # Linear increasing timestep for remainder
+    for i in 2:(num_stages - 2)
+        c[i] = c_s2 * (i - 1) / (num_stages - 3)
+    end
+
+    return c
+end
+
+function PERK3_butcher_tableau_objective_function(a_unknown, num_stages,
```

Comment:
```suggestion
function PERK3_Butcher_tableau_objective_function(a_unknown, num_stages,
```

I guess we can write names in capitals.